apps/game/src/ungap/trip/mod.rs

```diff
@@ -83,12 +83,14 @@ impl TripPlanner {
 
         let main_route = RouteDetails::main_route(ctx, app, self.waypoints.get_waypoints());
         self.main_route = main_route.details;
-        world
-            .add(ID::MainRoute)
-            .hitbox(main_route.hitbox)
-            .zorder(1)
-            .draw(main_route.draw)
-            .build(ctx);
+        if self.waypoints.get_waypoints().len() > 1 {
```

Comment:
![Screenshot from 2022-08-30 15-42-46](https://user-images.githubusercontent.com/1664407/187467437-d297f28f-558c-431c-8e28-fa46945eb4f2.png)
For routes with >= 3 waypoints, there are multiple hitboxes per route object in the `World`. It was most straightforward to just make `World` handle multiple hitboxes.